waveform_editor/derived_waveform.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, yaml_str, name, config, dd_version=None):
         self.dependencies = set()
         self.is_constant = False
         self.expression = None
+        self.user_symbols = {
+            name: obj for name, obj in vars(np).items() if isinstance(obj, np.ufunc)
+        }
```

Comment:
Could we make this a constant class variable?

Also, let's iterate over public numpy API only using `__all__`?
```python
NUMPY_UFUNCS = {}
for name in np.__all__:
    obj = getattr(np, name)
    if isinstance(obj, np.ufunc):
        NUMPY_UFUNCS[name] = obj
```